### PR TITLE
Hotfix release 0.266

### PR DESCRIPTION
**Pull request by souravpal**
Test plan :
(1) Unit test added
(2)End to end testing in DR Cluster

### CODE REVIEW

**Review by neeradsomanchi — APPROVED** [review]

File: presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/SimpleTtlNodeSelector.java [hunk 1]
```diff
@@ -227,8 +227,8 @@ public SplitPlacementResult computeAssignments(Set<Split> splits, List<RemoteTas
     private boolean isTtlEnough(ConfidenceBasedTtlInfo ttlInfo, Duration estimatedExecutionTime)
     {
         Instant expiryTime = ttlInfo.getExpiryInstant();
-        long timeRemaining = MILLIS.between(Instant.now(), expiryTime);
-        return new Duration(Math.max(timeRemaining, 0), TimeUnit.MILLISECONDS).compareTo(estimatedExecutionTime) >= 0;
+        long timeRemaining = SECONDS.between(Instant.now(), expiryTime);
```
Comment (ajaygeorge) [inline presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/SimpleTtlNodeSelector.java:230]: `timeRemainingInSeconds` will make it clearer

**Comment by ajaygeorge** [comment]
Can we add a test case and update the Title of the PR as well.
The fix needs to be added to master as well I presume?
Don't think we need a release note since this is not a user impacting change.

File: presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/SimpleTtlNodeSelector.java (hunk 1 above)
Comment (souravpal) [reply presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/SimpleTtlNodeSelector.java]: Done

**Comment by souravpal** [comment]
> Can we add a test case and update the Title of the PR as well. The fix needs to be added to master as well I presume? Don't think we need a release note since this is not a user impacting change.

Added unit test - yes we need to add the fix to the master as well. This fix required going forward.

**Review by tdcmeehan — APPROVED** [review]

**Review by ajaygeorge — APPROVED** [review]

**Pull request merged**